bugsnag-android-core/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -49,7 +49,7 @@
     private boolean automaticallyCollectBreadcrumbs = true;
 
     private boolean detectAnrs = false;
-    private boolean detectNdkCrashes = BuildConfig.DETECT_NDK_CRASHES;
+    private boolean detectNdkCrashes = false;
```

Comment:
Does this mean the NDK config option is no longer supported via the manifest?

Reply:
Nope, just that we always default it to false.

Reply:
Wait but why?

Reply:
This made sense with the original design where only `bugsnag-android` would have been published. Due to the fact we're now publishing bugsnag-android-ndk this will need to be updated.